packages/babel-helpers/README.md

```diff
@@ -19,3 +21,37 @@ const typeofHelper = helpers.get('typeof');
 t.isExpressionStatement(typeofHelper);
 // true
 ```
+
+Inside a plugin:
+
+```js
+export default {
+  visitor: {
+    UnaryExpression(path) {
+      // The .addHelper function adds, if needed, the helper to the file
+      // and returns an expression which references the helper
+      const typeofHelper = this.addHelper("typeof");
+      t.isExpression(typeofHelper); // true
+  }
+};
+```
+
+## How to define an helper?
+
+> **NOTE**: This package is only meant to be used by the packages inluded in this repository. There is no way for third-party plugins to define an helper.
+
+Helpers are defined in the `src/helpers.js` file.
+```js
```

Comment:
Nit: add a new line separating the code block from the line before it (not having it will often cause the website to not be formatted properly when it parses the READMEs)